vector/src/main/res/layout/item_bottom_sheet_message_preview.xml

```diff
@@ -103,4 +103,17 @@
         tools:text="1080 x 1024 - 43s - 12kB"
         tools:visibility="visible" />
 
+    <im.vector.app.features.location.MapTilerMapView
+        android:id="@+id/bottom_sheet_message_preview_location"
+        android:layout_width="0dp"
+        android:layout_height="200dp"
+        android:layout_marginTop="6dp"
+        android:importantForAccessibility="no"
```

Comment:
Not sure this is correct for a11y. Maybe add a contentDescription instead?

Reply:
Done as "Location".